File: GoodAPI.Data/Models/Medicine.cs
using System.ComponentModel.DataAnnotations;

namespace GoodAPI.Models
{
    public class Medicine
    {
        [Key]
        public int MedicineId { get; set; }          
        public string Name { get; set; }             
        public string GenericName { get; set; }      
        public string Manufacturer { get; set; }     
        public DateTime ExpiryDate { get; set; }    
        public decimal Price { get; set; }           
        public string Dosage { get; set; }           
        public string Form { get; set; }             
        public string Description { get; set; }      
        public bool PrescriptionRequired { get; set; }
    }
}

File: RegisterMedicinesFront/Models/Medicine.vb
Imports System.ComponentModel.DataAnnotations

Public Class Medicine
    <Key>
    Public Property MedicineId As Integer

    Public Property Name As String

    Public Property GenericName As String

    Public Property Manufacturer As String

    Public Property ExpiryDate As DateTime

    Public Property Price As Decimal

    Public Property Dosage As String

    Public Property Form As String

    Public Property Description As String

    Public Property PrescriptionRequired As Boolean
End Class
